src/components/map/map.scss

```diff
@@ -8,9 +8,17 @@
 gx-map {
   @include visibility(block);
   .gxMapContainer {
-    @include widthHeightsize(100%, 100%);
+    @include widthHeightsize(100%, 500px);
```

Comment:
I prefer to left it as it is now.
The name of the mixin explains exactly what it changes and the order of how the parameters must be passed, avoiding going to the mixin definition to remind how it works.
The only change that I would do is to call it as ```widthHeightSizes``` respecting the camel-case syntax.
Wht do u think?

Reply:
Fixed in 5bba66b

Reply:
I don't agree. Mixin's, like functions, should be named to concisely reflect what the code does. Parameters have names and orders of themselves, so you don't need to use them in the function's name.
I think that, instead of naming the function with the parameter's names, we should give the parameters better names, like `$width` and `$height` instead of `$valueX` and `$valueY`.

Reply:
"_I think that, instead of naming the function with the parameter's names, we should give the parameters better names, like $width and $height instead of $valueX and $valueY._"
Yeap. That I thought while I reading the first part of your answer. I liked so much. Applying your fixing advice 😉👌

Reply:
Fixed in 8ce6c08